src/bin/mmix-sim.rs

```diff
@@ -0,0 +1,50 @@
+extern crate mmix;
+
+use mmix::machine::Machine;
+
+use std::fs::File;
+use std::io::{Error, Read};
+
+fn main() {
```

Comment:
We don't need to put this code in `src/bin/mmix-sim.rs`. Better add a `src/main.rs` since a project can have both, a `lib.rs` and a `main.rs`.

[Explanation](http://doc.crates.io/guide.html#project-layout):
> - The default library file is src/lib.rs.
> - The default executable file is src/main.rs.